### PR TITLE
Pass verbosity level on to klog

### PR DESCRIPTION
**Pull request by coryrc**
This was left out of #1022

### CODE REVIEW

File: test/logging/logging.go [hunk 1]
```diff
@@ -159,6 +161,10 @@ func InitializeLogger() {
 		logger = zap.New(zapCore)
 		zap.ReplaceGlobals(logger) // Gets used by klog/glog proxy libraries
 
+		// Set klog/glog verbosities (works with and without proxy libraries)
+		klogLevel := klog.Level(0)
+		klogLevel.Set(fmt.Sprint(Verbosity))
```
Comment (vagababov) [inline test/logging/logging.go]: Verbosity.String()?

Comment (coryrc) [reply test/logging/logging.go]: I accidentally copied an old version, it shouldn't be capitalized.

It's an int and int doesn't have a .String() AFAICT.

Comment (vagababov) [reply test/logging/logging.go]: If it's an int, then just `strconv.Itoa`.

**Pull request merged**